d3a_api_client/setups/auto_offer_bid_on_device.py

```diff
@@ -35,7 +35,7 @@ def on_trade(self, trade_info):
 
 
 # Connects one client to the load device
-load = AutoOfferBidOnMarket('load', autoregister=True)
+# load = AutoOfferBidOnMarket('load', autoregister=True)
```

Comment:
Do we want to keep this as documentation, or is it a leftover?

Reply:
Dont know why this is there, will revert it (will be removed in 3398 anyway). Thanks!